drivers/everest/CMakeLists.txt

```diff
@@ -6,20 +6,20 @@ add_library(${everest_target}
   library/Hacl_Curve25519_joined.c)
 
 set_base_compile_options(${everest_target})
-target_include_directories(${everest_target}
-  PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
-         $<BUILD_INTERFACE:${TF_PSA_CRYPTO_DIR}/include>
-         $<BUILD_INTERFACE:${TF_PSA_CRYPTO_DIR}/drivers/builtin/include>
+  target_include_directories(${everest_target}
+  PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include/tf-psa-crypto/private>
```

Comment:
This doesn't look right. The point of this task is so that e.g. `#include "everest/everest.h"` no longer works, and instead you need `#include "tf-psa-crypto/private/everest/everest.h"`. So `private` should not be part of any directory on the include list. But `ecdh.h` should change.

Reply:
Oh I see. This was not something mentioned in the [original issue description](https://github.com/Mbed-TLS/TF-PSA-Crypto/issues/229) and I had to dig trough #235 [comments](https://github.com/Mbed-TLS/TF-PSA-Crypto/pull/235#discussion_r2073050303) to find a mention to it. Plus it's also market with the :rocket: so my thought was that everything was already addressed and I only had to take care of reorganizing PRs.
Apparently this is not the case and there's more changes required. I will try to address this as well

Reply:
I should have addressed this part now.